packaging/mlops-agent.spec

```diff
@@ -32,18 +32,18 @@
 %endif
 ###########################################################################
 # Package / sub-package definitions
-Name:		machine-learning-agent
-Summary:	AI service agent for ML(Machine Learning) API and NNStreamer
+Name:		mlops-agent
+Summary:	AI service agent for MLOps and ML(Machine Learning) API
 # Synchronize the version information for ML-Agent.
-# 1. Tizen   : ./packaging/machine-learning-agent.spec
+# 1. Tizen   : ./packaging/mlops-agent.spec
```

Comment:
need to update line 202 also (machine-learning-agent.spec > mlops-agent.spec)

Reply:
ACK!